pythoncalculator/divide.py

```diff
@@ -0,0 +1,2 @@
+def divide(x, z):
```

Comment:
Can you please change  the parameters "y" instead of "z"
```suggestion
def divide(x, y):
```